src/Microsoft.VisualStudio.VsInteractiveWindow/IVsInteractiveWindowFactory2.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.InteractiveWindow.Shell
+{
+    public interface IVsInteractiveWindowFactory2: IVsInteractiveWindowFactory 
+    {
+        IVsInteractiveWindow Create(Guid providerId, int instanceId, string title, IInteractiveEvaluator evaluator, IVsInteractiveWindowDecorator decorator, __VSCREATETOOLWIN creationFlags = 0);
```

Comment:
Can we move all of the parameters into a POD VsInteractiveWindowCreationParameters type class which just has getters/setters for the parameters?  That way we can add values to it in the future w/o needing to add new interfaces and not have a ton of parameters to pass in.